src/celeritas/phys/PhysicsStepView.hh

```diff
@@ -120,7 +120,7 @@ class PhysicsStepView
 /*!
  * Construct from shared and state data.
  */
-PhysicsStepView::PhysicsStepView(const PhysicsParamsRef& params,
+CELER_FUNCTION PhysicsStepView::PhysicsStepView(const PhysicsParamsRef& params,
```

Comment:
Is this really how `clang-format` did this?

Reply:
I might need to reformat :)